packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2840,11 +2866,9 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return { latestSnapshotRefSeq, latestSnapshotVersionId: versionId };
     }
 
-    private async fetchSnapshotFromStorage(
-        versionId: string | null,
+    private async fetchLatestSnapshotFromStorage(
         logger: ITelemetryLogger,
         event: ITelemetryGenericEvent,
-        fetchSource?: FetchSource,
```

Comment:
>FetchSource [](http://example.com/codeflow?start=22&length=11)

@jatgarg , There are only 2 call sites for fetchSnapshotFromStorage and we never use the versionId anyway. Do you see any problem in doing that?

Reply:
No problem in that as that would fetch the latest snapshot in that case.